### PR TITLE
Rename project directory #130

### LINKED ISSUE #130: Rename project directory
#### Problem description
The project directory needs to be renamed from `share_app` to `skunkworks_crow`

### PR DESCRIPTION
**Pull request by prajurock**
Closes #130 

#### Before submitting this PR, please make sure you have:
- [x] run `./gradlew pmd checkstyle lint findbugs` and confirmed all checks still pass OR confirm CircleCI build passes
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/share/blob/master/share_app/src/main/assets/open_source_licenses.html).

### CODE REVIEW

**Comment by prajurock** [comment]
@shobhitagarwal1612 @lakshyagupta21 Please review

**Comment by shobhitagarwal1612** [comment]
The circle ci build is failing. Please fix that first

**Comment by prajurock** [comment]
@shobhitagarwal1612 I am not getting what's the error. Please HELP

**Comment by shobhitagarwal1612** [comment]
Config.yml file is still referencing the old directory name. That needs to be fixed as well

**Comment by prajurock** [comment]
@shobhitagarwal1612 @lakshyagupta21 Please review

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#Tue Jun 12 21:47:54 IST 2018
+#Fri Mar 01 02:41:23 IST 2019
```
Comment (shobhitagarwal1612) [inline gradle/wrapper/gradle-wrapper.properties]: Please revert this change

File: skunkworks_crow/build.gradle [hunk 2]
```diff
@@ -6,7 +6,7 @@ android {
     defaultConfig {
         applicationId "org.odk.share"
         minSdkVersion 16
-        targetSdkVersion 22
+        targetSdkVersion 27
```
Comment (shobhitagarwal1612) [inline skunkworks_crow/build.gradle]: Please revert this unrelated change. This should be fixed separately as increasing the target version will have more changes too
